<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configurator</class>
 <widget class="QWidget" name="Configurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>492</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Configurator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/img/icon.svg</normaloff>:/img/icon.svg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="localWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#4C5844;
color:white;
border: 1px solid #7A836E;</string>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>44</y>
      <width>681</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>391</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2,40,46,20,3">
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="application.qrc">:/img/icon.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="windowName">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#899581;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>windowName</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #4C5844;
	color: #899581;
	border: 1px solid #7A836E;
}

QPushButton:hover {
    background-color: #3E4637;
}
</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="main_groupBox">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>45</y>
     <width>391</width>
     <height>445</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #7A836E;margin:10px;color:#D9D9D9;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="widgetPosition_groupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>181</width>
      <height>118</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 0px 0px 10px 10px;
border: 1px solid #7A836E;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="widgetPosition_radioButton__4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>71</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
	margin: 18px;
	border: 0px solid #7A836E;
	color: #D9D9D9;
}
QRadioButton:hover {
	color: #B8B857;
}
QRadioButton:checked {
	color: #B8B857;
}</string>
     </property>
     <property name="text">
      <string>1.1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="widgetPosition_radioButton__2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>4</y>
       <width>71</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
	margin: 18px;
	border: 0px solid #7A836E;
	color: #D9D9D9;
}
QRadioButton:hover {
	color: #B8B857;
}
QRadioButton:checked {
	color: #B8B857;
}</string>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="widgetPosition_radioButton__1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>4</y>
       <width>85</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
	margin: 18px;
	border: 0px solid #7A836E;
	color: #D9D9D9;
}
QRadioButton:hover {
	color: #B8B857;
}
QRadioButton:checked {
	color: #B8B857;
}</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="widgetPosition_radioButton__3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>85</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
	margin: 18px;
	border: 0px solid #7A836E;
	color: #D9D9D9;
}
QRadioButton:hover {
	color: #B8B857;
}
QRadioButton:checked {
	color: #B8B857;
}</string>
     </property>
     <property name="text">
      <string>1.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="widgetPosition_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid black;margin: 6px 0px 6px 7px;</string>
    </property>
    <property name="text">
     <string>Widget position</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="symbol_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>304</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 0px 0px 10px 10px;
background-color:#3E4637;</string>
    </property>
   </widget>
   <widget class="QLabel" name="symbol_textEdit_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid black;margin: 6px 0px 6px 7px;</string>
    </property>
    <property name="text">
     <string>Symbol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>393</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:#4C5844;
	color:#899581;
	border: 1px solid #7A836E;
	margin: 0px 0px 10px 10px;
}

QPushButton:hover {
    background-color: #3E4637;
}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>353</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:#4C5844;
	color:#899581;
	border: 1px solid #7A836E;
	margin: 0px 0px 10px 10px;
}

QPushButton:hover {
    background-color: #3E4637;
}
</string>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="chooseTextColor_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>157</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid black;margin: 6px 0px 6px 7px;</string>
    </property>
    <property name="text">
     <string>Select text color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectColor_button">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>197</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:#4C5844;
	color:#899581;
	border: 1px solid #7A836E;
	margin: 0px 0px 10px 10px;
}

QPushButton:hover {
    background-color: #3E4637;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="application.qrc">
      <normaloff>:/img/color-dropper.svg</normaloff>:/img/color-dropper.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="currentColor_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>197</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 0px 0px 10px 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="protocol_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid black;margin: 6px 0px 6px 7px;</string>
    </property>
    <property name="text">
     <string>Protocol</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="protocol_groupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>269</y>
      <width>181</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#4C5844;
border: 1px solid #7A836E;
margin: 0px 0px 10px 10px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="protocol_https_radioButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>12</y>
       <width>71</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
	margin:0px 0px 0px 8px;
	border: 0px solid #7A836E;
	background-color: rgba(255, 255, 255, 0);
	color: #D9D9D9;
}
QRadioButton:hover {
	color: #B8B857;
}
QRadioButton:checked {
	color: #B8B857;
}</string>
     </property>
     <property name="text">
      <string>HTTP/S</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="protocol_ws_radioButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>94</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
	margin:0px 0px 0px 8px;
	border: 0px solid #7A836E;
	background-color: rgba(255, 255, 255, 0);
	color: #D9D9D9;
}
QRadioButton:hover {
	color: #B8B857;
}
QRadioButton:checked {
	color: #B8B857;
}</string>
     </property>
     <property name="text">
      <string>WebSocket</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dev_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>353</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#4C5844;
border: 1px solid #7A836E;
margin: 0px 0px 10px 10px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="auto_update_checkBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>6</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	border: 0px solid black;
	margin: 0px 0px 0px 12px;
	background-color: rgba(255, 255, 255, 0);
}
QCheckBox:hover {
	color: #B8B857;
}
QCheckBox:checked {
	color: #B8B857;
}</string>
     </property>
     <property name="text">
      <string>Auto-Update</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="use_binance_us_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	border: 0px solid black;
	margin: 0px 0px 0px 12px;
	background-color: rgba(255, 255, 255, 0);
}
QCheckBox:hover {
	color: #B8B857;
}
QCheckBox:checked {
	color: #B8B857;
}</string>
     </property>
     <property name="text">
      <string>Binance.US API</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
